detection_rules/rule.py

```diff
@@ -1159,11 +1159,14 @@ def flattened_dict(self) -> dict:
         flattened.update(self.metadata.to_dict())
         return flattened
 
-    def to_api_format(self, include_version=True) -> dict:
+    def to_api_format(self, include_version: bool = True, include_metadata: bool = False) -> dict:
```

Comment:
nit: any reason why you renamed from add_metadata to include_metadata between the functions?

Reply:
Started with command...ended up up in `to_api_format` with has a `include_version`. I'll keep it consistent with `include_metadata`